Source/SwiftLintFramework/Rules/UnusedOptionalBindingRule.swift

```diff
@@ -10,7 +10,7 @@ import Foundation
 import SourceKittenFramework
 
 public struct UnusedOptionalBindingRule: ASTRule, ConfigurationProviderRule {
-    public var configuration = SeverityConfiguration(.warning)
+    public var configuration = UnusedOptionalBindingConfiguration(ignoreOptionalTry: true)
```

Comment:
Since people are questioning this rule in general, I thought it'd make sense to make it more "permissive" by default but at same time allowing people to keep the old behavior.